### DISCUSSION TITLE
Fitur Kode Wilayah bagaimana?

### DISCUSSION THREAD

**Discussion by IrsyadProject** [ivanwilliammd/satusehat-integration#136] [Ideas]
Untuk fitur kode wilayah bagaimana cara menjalankannya ya? soalnya di dokumentasinya tidak ada, terimakasih

**Comment by IrsyadProject** [comment]
dibagian SatusehatIntegrationServiceProvider kurang publish kode wilayah indonesia

**Comment by ivanwilliammd** [comment]
Closed on https://github.com/ivanwilliammd/satusehat-integration/pull/57
